client/src/app/__tests__/EmptyTabSpec.js

```diff
@@ -65,29 +61,20 @@ describe('<EmptyTab>', function() {
       sinon.stub(Flags, 'get').withArgs(DISABLE_DMN).returns(true);
 
       // when
-      const {
-        tree
-      } = createEmptyTab();
+      createEmptyTab();
 
       // then
-      expect(tree.findWhere(
-        wrapper => wrapper.text().startsWith('DMN diagram')).first().exists()).to.be.false;
+      expect(screen.queryAllByText('DMN diagram')).to.be.empty;
```

Comment:
Cf. pattern exposed here: https://github.com/camunda/camunda-modeler/pull/5148/files#diff-9b1beed3f75963d3c20b373ee35aa9a6d51258e4ca4ee4220b4998e8065ec30fR71

TLDR: There is generally no need to use `screen`.